cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -808,6 +816,44 @@ instance ToObject NtN.HandshakeTr where
              , "bearer" .= show b
              , "event" .= show ev ]
 
+instance ( Mock.SimpleCrypto c
+         , Typeable a)
+ => ToObject (Header (Mock.SimpleBlock c a)) where
+  toObject _verb b =
+    mkObject $
+        [ "kind" .= String "SimpleBlockHeader"
+        , "hash" .= condense (blockHash b)
+        , "prevhash" .= condense (blockPrevHash b)
+        , "slotNo" .= condense (blockSlot b)
+        , "blockNo" .= condense (blockNo b) ]
+
+instance ToObject (Header ByronBlock) where
+  toObject _verb b =
+    mkObject $
+        [ "kind" .= String kind
+        , "hash" .= condense (blockHash b)
+        , "prevhash" .= condense (blockPrevHash b)
+        , "slotNo" .= condense (blockSlot b)
+        , "blockNo" .= condense (blockNo b)
+        ] <>
+        case view of
+          Nothing -> []
+          Just (diffic, deleg) ->
+            [ "difficulty" .= toJSON (Common.unChainDifficulty diffic)
```

Comment:
The chain difficulty should be exactly the same thing as block number, so I'd remove the chain difficulty in favour of the block number.

Reply:
@mrBliss, done, thank you!